modules/adpod.js

```diff
@@ -457,7 +467,27 @@ export function getTargeting({codes, callback} = {}) {
 
   let bids = getBidsForAdpod(bidsReceived, adPodAdUnits);
   bids = (competiveExclusionEnabled || deferCachingEnabled) ? getExclusiveBids(bids) : bids;
-  bids.sort(sortByPricePerSecond);
+
+  let prioritizeDeals = config.getConfig('adpod.prioritizeDeals');
+  if (prioritizeDeals) {
+    let [otherBids, highPriorityDealBids] = bids.reduce((partitions, bid) => {
+      let bidDealTier = utils.deepAccess(bid, 'video.dealTier');
+      let minDealTier = config.getConfig(`adpod.dealTier.${bid.bidderCode}.minDealTier`);
+      if (minDealTier && bidDealTier) {
+        partitions[Number(bidDealTier > minDealTier)].push(bid);
```

Comment:
What happens here when you case the `bidDealTier` as a `Number` when a publisher defined a prefix?  Could that skew the comparison when you have `(tier7 > 5)`?

Reply:
bid.video.dealTier is an int value which all bidders need to abide.